$NetBSD: patch-aa,v 1.25 2005/01/17 14:54:01 atatat Exp $

If LSOF_MORE_SECURE is non-zero, compile lsof with security, and
only let unprivileged (non-root) users view their own open files.
Set LSOF_LESS_SECURE_SOCKETS to non-zero in order to allow non-root
users to see open sockets, even when LSOF_MORE_SECURE is set.

--- dialects/n+obsd/machine.h.orig	2004-10-17 17:55:27.000000000 -0400
+++ dialects/n+obsd/machine.h
@@ -383,7 +383,9 @@
  * (the one that its user logged on with) of the lsof process.
  */
 
-/* #define	HASSECURITY	1 */
+#if LSOF_MORE_SECURE
+#define	HASSECURITY	1
+#endif
 
 
 /*
@@ -392,7 +394,9 @@
  * listing is selected by the "-i" option.
  */
 
-/* #define	HASNOSOCKSECURITY	1	*/
+#if LSOF_LESS_SECURE_SOCKETS
+#define	HASNOSOCKSECURITY	1
+#endif
 
 
 /*
